src/rpc/mempool.cpp

```diff
@@ -91,11 +93,14 @@ static RPCHelpMan sendrawtransaction()
             std::string err_string;
             AssertLockNotHeld(cs_main);
             NodeContext& node = EnsureAnyNodeContext(request.context);
+            const auto method = gArgs.GetBoolArg("-privatebroadcast", DEFAULT_PRIVATE_BROADCAST)
```

Comment:
In cf76f4f0fe1a3ef0730f70a035c10e01712b6d6b: Shouldn't we check here whether any private network is reachable? Because we can start a node with `-privatebroadcast` with `NET_ONION` becoming reachable later, no?

Reply:
Done.